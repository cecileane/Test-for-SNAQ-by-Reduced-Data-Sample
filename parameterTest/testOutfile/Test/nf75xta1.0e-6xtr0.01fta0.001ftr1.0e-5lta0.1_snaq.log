optimization of topology, BL and inheritance probabilities using:
 hmax = 1,
 tolerance parameters: ftolRel=1.0e-5, ftolAbs=0.001,
                       xtolAbs=1.0e-6, xtolRel=0.01.
 max number of failed proposals = 75, liktolAbs = 0.1.
rootname for files: nf75xta1.0e-6xtr0.01fta0.001ftr1.0e-5lta0.1_snaq
BEGIN: 2 runs on starting tree (2,3,(4,(5,(1,6):0.386):1.19):0.028);

main seed 34495
seed: 34495 for run 1
Sat Oct  1 17:13:44 2016

 BEGIN SNaQ for run 1, seed 34495 and hmax 1 changed starting topology by NNI move

Begins heuristic optimization of network------

found best network, now we re-optimize branch lengths and gamma more precisely
STOPPED by number of failures criteria
END optTopLevel: found minimizer topology at step 138 (failures: 75) with -loglik=165.73227 and ht_min=[0.32794,0.0,0.87065,10.0,0.48584]
PERFORMANCE: total number of moves (proposed, successful, accepted) in general, and to fix gamma=0.0,t=0.0 cases
	--------moves general--------				 --------moves gamma,t------	
move	 Num.Proposed	 Num.Successful	 Num.Accepted	 | Num.Proposed	 Num.Successful	 Num.Accepted
add		 1		 1		 1	 |	 NA 		 NA 		 NA 
mvorigin	 39		 39		 2	 |	 0		 0		 --
mvtarget	 33		 33		 0	 |	 0		 0		 --
chdir		 12		 12		 0	 |	 1		 1		 --
delete		 0		 0		 0	 |	 0		 0		 --
nni		 53		 32		 1	 |	 8		 8		 --
Total		 138		 117		 4	 |	 9		 9		 7
Proportion	 -- 		 1.0		 0.0	 |	 -- 		 1.0		 0.8

 FINISHED SNaQ for run 1, -loglik of best 165.73226941153675
((4,(1,2):0.4858404700039288):0.0,((5,#H7:::0.3279371088510262):10.0,6):0.8706545839692994,(3)#H7:::0.6720628911489738);
---------------------
seed: 69842 for run 2
Sat Oct  1 17:13:52 2016

 BEGIN SNaQ for run 2, seed 69842 and hmax 1
Begins heuristic optimization of network------

found best network, now we re-optimize branch lengths and gamma more precisely
STOPPED by number of failures criteria
END optTopLevel: found minimizer topology at step 88 (failures: 75) with -loglik=128.22643 and ht_min=[0.30771,0.66107,0.4599,8.87903,1.73912]
PERFORMANCE: total number of moves (proposed, successful, accepted) in general, and to fix gamma=0.0,t=0.0 cases
	--------moves general--------				 --------moves gamma,t------	
move	 Num.Proposed	 Num.Successful	 Num.Accepted	 | Num.Proposed	 Num.Successful	 Num.Accepted
add		 2		 2		 1	 |	 NA 		 NA 		 NA 
mvorigin	 24		 24		 0	 |	 0		 0		 --
mvtarget	 25		 25		 0	 |	 0		 0		 --
chdir		 4		 4		 0	 |	 1		 1		 --
delete		 0		 0		 0	 |	 0		 0		 --
nni		 33		 24		 0	 |	 3		 3		 --
Total		 88		 79		 1	 |	 4		 4		 4
Proportion	 -- 		 1.0		 0.0	 |	 -- 		 1.0		 1.0

 FINISHED SNaQ for run 2, -loglik of best 128.2264279477359
(((1,#H7:::0.30771399349759593):8.879032389718937,2):0.6610716895952385,4,((5,(6)#H7:::0.6922860065024041):1.7391201383329322,3):0.45989574697700103);
---------------------

Sat Oct  1 17:14:00 2016
MaxNet is (((1,#H7:::0.30771399349759593):8.879032389718937,2):0.6610716895952385,4,((5,(6)#H7:::0.6922860065024041):1.7391201383329322,3):0.45989574697700103); 
with -loglik 128.2264279477359
