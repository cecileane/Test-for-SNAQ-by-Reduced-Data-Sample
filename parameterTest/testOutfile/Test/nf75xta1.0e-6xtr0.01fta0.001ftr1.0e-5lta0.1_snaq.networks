(((1,#H7:::0.30771399349759593):8.879032389718937,2):0.6610716895952385,4,((5,(6)#H7:::0.6922860065024041):1.7391201383329322,3):0.45989574697700103);, with -loglik 128.2264279477359 (best network found, remaining sorted by log-pseudolik; the smaller, the better)
(((1)#H7:::0.6476717897801069,2):2.6978324373792977,4,((5,(6,#H7:::0.35232821021989313):1.2218435076627439):0.4380989863426027,3):0.5531391390465741);, with -loglik 129.88692298001055
(#H7:::0.38383310113223845,(1,(6,(5,((4)#H7:::0.6161668988677615,3):0.01972006735982483):0.2210635549345803):1.1759655597077139):7.845588828841086,2);, with -loglik 171.9259732420138
(#H7:::0.406808277209181,4,((5,(6,((2)#H7:::0.593191722790819,1):3.2374381182992797):0.153716658714338):0.020180305824623452,3):2.2091762346758816);, with -loglik 192.6562352328175
(((1,(6,(5,#H7:::0.46409275198641253):1.3186028106750776):1.0178097788658396):0.10905260847522803,2):0.11158678171959983,4,(3)#H7:::0.5359072480135875);, with -loglik 214.7624686361291
(((1,(6,(5)#H7:::0.5111361359208159):3.9893939302124597):0.038899333299203205,2):0.2128344760362582,4,(#H7:::0.48886386407918414,3):1.1390357765491834);, with -loglik 232.0495199019071
