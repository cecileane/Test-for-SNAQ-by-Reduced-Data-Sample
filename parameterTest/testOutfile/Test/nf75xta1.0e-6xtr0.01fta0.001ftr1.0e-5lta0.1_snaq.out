(((1,#H7:::0.30771399349759593):8.879032389718937,2):0.6610716895952385,4,((5,(6)#H7:::0.6922860065024041):1.7391201383329322,3):0.45989574697700103); -Ploglik = 128.2264279477359
 Dendroscope: (((1,#H7):8.879032389718937,2):0.6610716895952385,4,((5,(6)#H7):1.7391201383329322,3):0.45989574697700103);
 Elapsed time: 16.014767017 seconds in 2 successful runs
-------
List of estimated networks for all runs (sorted by log-pseudolik; the smaller, the better):
 (((1,#H7:::0.30771399349759593):8.879032389718937,2):0.6610716895952385,4,((5,(6)#H7:::0.6922860065024041):1.7391201383329322,3):0.45989574697700103);, with -loglik 128.2264279477359
 ((4,(1,2):0.4858404700039288):0.0,((5,#H7:::0.3279371088510262):10.0,6):0.8706545839692994,(3)#H7:::0.6720628911489738);, with -loglik 165.73226941153675
-------
